b2sdk/bucket.py

```diff
@@ -796,9 +805,10 @@ def concatenate(
         :param bool legal_hold: legal hold setting
         :param int min_part_size: lower limit of part size for the transfer planner, in bytes
         :param int max_part_size: upper limit of part size for the transfer planner, in bytes
+        :param str,None large_file_sha1: SHA-1 hash of the result file or ``None`` if unknown
         """
         return self.create_file(
-            WriteIntent.wrap_sources_iterator(outbound_sources),
+            list(WriteIntent.wrap_sources_iterator(outbound_sources)),
```

Comment:
I don't really like this change. Having iterator here is much nicer.

Reply:
I found no explanation, will remove.

Reply:
I found it now. `create_file` (and `emerge` below it) assumes that this will be a list.